auto-diagram/practice.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I presume this is a temporary playbook? Once completed, it probably needs a better name than `practice` and should use an inventory instead of hardcoded values.